src/dc/nnmf.js

```diff
@@ -44,6 +44,24 @@ export default class NNMF {
     }
     return (error);
   }
+  get positivity() {
```

Comment:
This function should be moved to the test case. The point of NNMF being to always produce a positive matrix, this information does not seem useful in the class itself.

Reply:
Thank you, done.